src/wallet/wallet2.cpp

```diff
@@ -493,22 +496,31 @@ size_t estimate_rct_tx_size(int n_inputs, int mixin, int n_outputs, size_t extra
   // ecdhInfo
   size += 2 * 32 * n_outputs;
   // outPk - only commitment is saved
-  size += 32 * n_outputs;
+  if (!bulletproof)
+    size += 32 * n_outputs;
```

Comment:
Oh, does the above calculation for BP include the commitment itself, too? Then my calculation gets closer (704). Actually I don't know where that remaining 3 bytes come from.

In this case, isn't it simpler to keep this line unchanged and change the above calculation to:

    size += (672 + 3) * n_outputs;

so that both Borromean & BP cases become consistent?

Reply:
I was originally saving V, and omitting these. This was changed since (easier for pruning) so I changed this to what you said. The 3 bytes are vector sizes (varint).